### ISSUE TITLE
Overwritten file with same name

### ISSUE THREAD

**Issue by radeno** [10up/windows-azure-storage#81]
Hello,

I think i have found critical bug with this plugin. Our client uploads a lot of files to WP site which are stored on Azure Storage.

When client uploaded file with same file name within same path then file is overwritten.

For example 2019/06/2.jpg and after some days (in same month) is uploaded another file with same name then original file is overwritten.
Original Wordpress implementation checks if file exists and then append number suffix (-1, -2, -3, -4, etc)

I think this Azure storage implementation doesn't check for existing files. Does it?

This is very ugly issue.

**Comment by andrewleader** [comment]
Yeah I have the same problem too. 4.2.0 release branch hasn't fixed this.
